src/main/java/com/fvv/bookstore/exception/BookValidationException.java

```diff
@@ -0,0 +1,26 @@
+package com.fvv.bookstore.exception;
```

Comment:
since you now have 2 specific exceptions for "Book" and assuming you will have others for the other beans, please add a new package here...you will need to move the classes for this new package and change the references. 

new package: `com.fvv.bookstore.exception.book`